src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -1,6 +1,18 @@
 package org.jenkinsci.pipeline_steps_doc_generator;
 
-import hudson.model.Descriptor;
+import java.io.IOException;
```

Comment:
Also wanted to reiterate here because of the same movements: normally when editing code, you don't want to move around dependency imports.  A lot of projects will have a standard of imports which make it easier to quickly look at what's included in a project.  I looked at Jenkins and didn't see anything